server/webserver.go

```diff
@@ -750,6 +753,25 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 		w.Write([]byte(fmt.Sprintf("\n\nLatestPlaylist: %v", s.LatestPlaylist())))
 	})
 
+	mux.HandleFunc("/logLevel", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
As mentioned earlier, the convention we have for the CLI endpoints is to have explicit getters / setters. So for consistency, can we split this up into two functions: `getLogLevel` and `setLogLevel` .

Reply:
fixed